src/databricks/labs/dqx/llm/utils.py

```diff
@@ -0,0 +1,36 @@
+import inspect
+from typing import Any
+
+from databricks.labs.dqx.engine import DQEngineCore
+from databricks.labs.dqx.rule import CHECK_FUNC_REGISTRY
+
+
+def get_check_function_definition(custom_check_functions: dict[str, Any] | None = None) -> list[dict[str, str]]:
+    """A utility function to get the definition of all check functions.
+    This function is primarily used to generate a prompt for the LLM to generate check functions.
+
+    :param custom_check_functions: A dictionary of custom check functions.
+        If provided, the function will use the custom check functions to resolve the check function.
+        If not provided, the function will use only the built-in check functions.
+
+    Returns:
+      list[dict]: A list of dictionaries, each containing the definition of a check function.
+    """
+    function_docs: list[dict[str, str]] = []
+    for name, func_type in CHECK_FUNC_REGISTRY.items():
+        func = DQEngineCore.resolve_check_function(name, custom_check_functions, fail_on_missing=True)
```

Comment:
```suggestion
        func = DQEngineCore.resolve_check_function(name, custom_check_functions, fail_on_missing=False)
```

If you fail on missing, exception will be thrown and the function execution will stop

Reply:
Changed to false